db/crud.go

```diff
@@ -1644,7 +1644,7 @@ func (db *Database) getChannelsAndAccess(doc *Document, body Body, revID string)
 		// No ChannelMapper so by default use the "channels" property:
 		value := body["channels"]
 		if value != nil {
-			array := base.ValueToStringArray(value)
+			array, _ := base.ValueToStringArray(value)
```

Comment:
This default sync function still silently ignores non-string channels. We should keep both consistent.

![Screenshot 2020-02-28 at 11 38 55](https://user-images.githubusercontent.com/1525809/75546096-becfd900-5a1f-11ea-9d02-a9c239569966.png)

Reply:
Fixed in latest version.